includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -66,9 +66,10 @@ public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 
 			case 'type' :
-				$value = ( 'primary' === $item['type'] )
-					? esc_html_e( 'Primary',   'easy-digital-downloads' )
-					: esc_html_e( 'Secondary', 'easy-digital-downloads' );
+				$value = ucwords( edd_get_address_type_label( $item['type'] ) );
```

Comment:
Just to clarify: as I read the code, if the address type is something other than `billing`, but the type has not been registered with the `edd_address_type_labels` filter, it will still show as "Billing" here. Is that the desired behavior?

Asking because in testing another issue, I added an address to the database with a different `type`, but it still displays as "Billing".

Reply:
Fixed that up @robincornett. I've set a fallback to just use the label that was passed in if it it's registered. If it is, it'll use it.

Reply:
Yes, it used the one I hard coded in.